### PR TITLE
Update hubs_tos.md

### PR DESCRIPTION
**Pull request by mafeld**

### CODE REVIEW

File: en/hubs_tos.md [hunk 1]
```diff
@@ -1,22 +1,17 @@
 # Terms of Service for Hubs
 
-Version 1.0, Effective October 11, 2022
-{: datetime="2022-10-11" }
+Version 1.0, Effective October 25, 2022
```
Comment (peiying2) [inline en/hubs_tos.md:3]: Should we update the version number?

Comment (mafeld) [reply en/hubs_tos.md:3]: no need. We never launched version 1. Just delayed it.

File: en/hubs_tos.md [hunk 2]
```diff
@@ -38,10 +33,14 @@ Any ideas, suggestions, and feedback about Hubs that you provide to us are entir
...
+
 ## 3. Conditions of Use
 By using Hubs, you agree that you will comply with Mozilla’s [Conditions of Use](https://www.mozilla.org/about/legal/acceptable-use/). Mozilla reserves the right to remove any content, suspend any users, and shut down any Hub or individual room it reasonably believes has violated these conditions.
 
-__Eligibility.__ To use the paid subscription service, you must be in a country where Hubs is available. Hubs is currently available in the United States, Canada, and the United Kingdom.
+__Eligibility.__ To use the paid subscription service, you must be in a country where Hubs is available. Hubs is currently available in the United States, Canada, Germany, and the United Kingdom.
```
Comment (peiying2) [inline en/hubs_tos.md:43]: Nit: do we want to list the countries in alphabetical order?

Comment (mafeld) [reply en/hubs_tos.md:43]: yes. I think that makes sense.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**